In-house patch to change the default for the heat_config_dir to
something less Windows-oriented.

Upstream bug https://bugs.launchpad.net/cloudbase-init/+bug/1633235 has
been filed to track this effort.

--- cloudbase-init-1.1.0/cloudbaseinit/conf/default.py
+++ cloudbase-init-1.1.0/cloudbaseinit/conf/default.py
@@ -125,7 +125,7 @@ class GlobalOptions(conf_base.Options):
                 help='Renames the builtin admin user instead of creating a '
                      'new user'),
             cfg.StrOpt(
-                'heat_config_dir', default='C:\\cfn',
+                'heat_config_dir', default='/system/volatile',
                 help='The directory where the Heat configuration files must '
                      'be saved'),
             cfg.BoolOpt(
